pyproject.toml

```diff
@@ -70,6 +70,7 @@ isort = "^5.13.2"
 openapi-spec-validator = "0.2.8"
 disutils = "^1.4.32.post2"
 setuptools = "^75.8.0"
+proto-schema-parser = "^1.5.0"
```

Comment:
[proto-schema-parser](https://github.com/criccomini/proto-schema-parser/) is a minimal package for parsing .proto, pure python, only dependency is Antlr.
```
  - Installing antlr4-python3-runtime (4.13.2)
  - Installing proto-schema-parser (1.6.0)
  ```